src/components/screens/NewsItem.js

```diff
@@ -113,7 +124,19 @@ export const NewsItem = ({ data }) => {
             !!mediaContent.sourceUrl.url &&
             section.push(
               <WrapperHorizontal key={`${index}-${contentBlock.id}-mediaContent${mediaContent.id}`}>
-                <HtmlView html={trimNewLines(mediaContent.sourceUrl.url)} />
+                <WebView
+                  source={{ html: trimNewLines(mediaContent.sourceUrl.url) }}
+                  style={styles.iframeWebView}
+                  scrollEnabled={false}
+                  bounces={false}
+                  injectedJavaScript={INJECTED_JAVASCRIPT_FOR_IFRAME_WEBVIEW}
```

Comment:
''for correct display with zoomlevel, a hacky javascript include is necessary'' is this the hacky js that you mentioned ? what is it ?

Reply:
yes, that constant `INJECTED_JAVASCRIPT_FOR_IFRAME_WEBVIEW` contains the "hacky javascript"